aat/src/resources/features/F-018/S-001.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "should invoke activity service And return response on successful requests",
+  "_guid_": "S-001",
+  "_extends_": "F-018_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is to add an activity for the case created above",
+    "contains a case Id that has just been created as in Standard_Full_Case_Creation_Data"
+  ],
+
+
+    "expectedResponse": {
+      "responseCode": 201,
```

Comment:
Expected response should extend a base object. Pls use inheritance as far as possible to eliminate redundant specification of common test data.

Reply:
updated